core/modules/loader/ClientConfig.h

```diff
@@ -86,7 +88,14 @@ class ClientConfig : public ConfigBase {
     /// Maximum number of insert requests allowed in the DoList.
     ConfigElement::Ptr _maxInserts{
         ConfigElement::create(cfgList, header, "maxInserts", ConfigElement::INT, false, "90000")};
-
+    /// When reaching maxInserts or maxLookups, sleep this long before trying to add more,
+    /// in micro seconds. 100000micro = 0.1sec
+    ConfigElement::Ptr _maxRequestSleepTime{
+        ConfigElement::create(cfgList, header,
+                              "maxRequestSleepTime", ConfigElement::INT, false, "100000")};
```

Comment:
Eventually, that value gets passed to util::ConfigStore::get() which takes a string for the default argument, and jumping through the hoops to take an integer here only to convert to a string later doesn't seem worth it at this time. 
I really want to avoid changing code in qserv proper until this branch is ready to merge to master.